src/shared/mentionable_text_input.rs

```diff
@@ -620,23 +646,39 @@ impl MentionableTextInput {
                     log!("Could not get client for @room avatar");
                 }
 
-                // If unable to display room avatar, show letter R with red background
+                // If unable to display room avatar, show first character of room name
                 if !room_avatar_shown {
-                    avatar_ref.show_text(cx, Some(COLOR_UNKNOWN_ROOM_AVATAR), None, "R");
+                    avatar_ref.show_text(cx, Some(COLOR_UNKNOWN_ROOM_AVATAR), None, &room_name_first_char);
                 }
 
+                // Apply the same height styling as user items for consistency
+                if is_desktop {
+                    room_mention_item.apply_over(
+                        cx,
+                        live!(
+                            height: (DESKTOP_ITEM_HEIGHT),
+                        ),
+                    );
+                } else {
+                    room_mention_item.apply_over(
+                        cx,
+                        live!(
+                            height: (MOBILE_ITEM_HEIGHT),
+                        ),
+                    );
+                }
```

Comment:
try to avoid duplicated code like this:

```suggestion
                let new_height = if is_desktop { DESKTOP_ITEM_HEIGHT } else { MOBILE_ITEM_HEIGHT };
                room_mention_item.apply_over(cx, live! {
                    height: (DESKTOP_ITEM_HEIGHT),
                });
```